src/css/custom.css

```diff
@@ -5,20 +5,20 @@
  * work well for content-centric websites.
  */
 
- html {
+html {
```

Comment:
Prettier made some formatting changes to this file... the only relevant change is the the font color for badges.